executor/common_fuchsia.h

```diff
@@ -176,13 +176,20 @@ long syz_mmap(size_t addr, size_t size)
 		fail("zx_object_get_info(ZX_INFO_VMAR) failed: %d", status);
 	zx_handle_t vmo;
 	status = zx_vmo_create(size, 0, &vmo);
-	if (status != ZX_OK)
+	if (status != ZX_OK) {
+		debug("zx_vmo_create failed with: %d", status);
 		return status;
+	}
+	status = zx_vmo_replace_as_executable(vmo, ZX_HANDLE_INVALID, &vmo);
+	if (status != ZX_OK)
+		fail("zx_vmo_replace_as_executable failed with: %d", status);
```

Comment:
This must not fail like ever-ever? With any insane inputs and process state?
If this will fail, it will bring down whole VMs and file bugs.

If this must not fail during the initial data map, but can fail when invoked by fuzzer with insane args, then return status here. os_init will check it and fail. Fuzzer will ignore failure just like it ignores failures of all other syscalls.